aws/modules/alb/variables.tf

```
@@ -0,0 +1,36 @@
+variable "name" {
+  description = "The name to use for all the cluster resources"
+  type        = string
+}
+
+variable "vpc_id" {
+  description = "The id of the vpc"
+  type        = string
+}
+
+variable "subnets" {
+  description = "The id of the subnets to associate with the ALB"
+  type        = list(string)
+}
+
+variable "security_groups" {
+  description = "The id of security groups to associate with the ALB"
+  type        = list(string)
+}
+
+variable "listener_port" {
+  description = "Value of the port for the listener"
+  type        = number
+}
+
+variable "listener_fixed_response" {
+  description = "Map of the response from the listener"
```

```suggestion
  description = "Default response if no rule matches"
```

maybe we should rename it to `listener_default_response`?